packages/components/src/utils/theme-variables.scss

```diff
@@ -6,3 +6,20 @@
 $components-color-accent: var(--wp-components-color-accent, var(--wp-admin-theme-color, #007cba));
 $components-color-accent-darker-10: var(--wp-components-color-accent-darker-10, var(--wp-admin-theme-color-darker-10, #006ba1));
 $components-color-accent-darker-20: var(--wp-components-color-accent-darker-20, var(--wp-admin-theme-color-darker-20, #005a87));
+
+// Used when placing text on the accent color.
```

Comment:
We should probably add documentation to the README:

- for the additional `background` prop that the `Theme` component accepts
- for all of the "output" colors that the theme generates, and their intended usage (like, for example, this comment)

When doing this, we should also understand what part of this should be marked as a "public API" of the component.

We should probably also explicitly write that these CSS vars (or context vars, in case we use react context) are not meant to be consumed outside of the package (unless we think they should?)

Reply:
Readme updated in b490e41ad1d552f2afcfe2618afbb13303270478

> are not meant to be consumed outside of the package (unless we think they should?)

Once this is production-ready, the CSS vars will in fact need to be used by components outside of wp-components if they want to participate in theming. Most of it will be automatic, if the custom components are built using wp-components, but some of them will have custom elements that need to be colored explicitly.